pkg/kubectl/cmd/run.go

```diff
@@ -213,7 +213,14 @@ func Run(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer, cmd *cob
 		if err != nil {
 			return err
 		}
-		return handleAttachPod(f, client, attachablePod, opts)
+		err = handleAttachPod(f, client, attachablePod, opts)
+		if err != nil {
+			return err
+		}
+		if opts.TTY && restartPolicy == api.RestartPolicyAlways {
+			fmt.Fprintf(cmdOut, "Session ended, resume using '$ kubectl attach %s -i -t' command when the pod is running\n", attachablePod.Name)
```

Comment:
The pod that created in run command doesn't have more than one container,  I think -c CONTAINER isn't needed.